### PR TITLE
More GitHub content 

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**

This PR adds additional topics: `git status`, `git remote`, and `git push`.

### CODE REVIEW

**Comment by brianamarie** [comment]
@github/services-implementation-engineers , @github/services-programs , if you have a chance to review or even directly edit, your help would be appreciated ✨ 

cc @sandhub @AmandaSwan @Chubbmo @tobiasahlin

File: git-guides/git-remote.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+If you try running `git remote -v` in your repositories, you'll probably see something called `origin`. You may notice `origin` in many messages from Git. `origin` is the human-friendly name for the URL that the remote repository is stored at. It's like a key value pair, and `origin` is the default.
+
+### What is `upstream`?
+
+You may need or want to work with multiple remotes for one local repository. This can be common in open source, when a contributor needs to create a fork of a repository to have permission to push changes to the remote.
+
+In this case, it's common to create and clone a fork. Then, the default remote would be `origin`, in reference to the fork. To make it easier to pull any changes to update the local copy of the fork from the original repository, many people add the original repository as a remote also. It's typical to name this remote `upstream`.
+
+![](https://githubtraining.github.io/training-manual/img/fork-to.png)
```
Comment (a-a-ron) [inline git-guides/git-remote.md]: We should update this image to match the look and feel of the other ✨ images

Comment (brianamarie) [reply git-guides/git-remote.md]: Very good point! I will bring this up in the slack channel to see if we can get this image updated.

File: git-guides/git-push.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+After you make and commit changes locally, you can share them with the remote repository using `git push`. Pushing changes to the remote makes your commits accessible to others who you may be collaborating with. This will also update any open pull requests with the branch that you're working on. 
+
+As best practice, it's important to run the `git pull` command before you push any new changes to the remote branch. This will update your local branch with any new changes that may have been pushed to the remote from other contributors. Pulling before you push can reduce the amount of merge conflicts you create on GitHub - allowing you to resolve them locally before pushing your changes to the remote branch. 
+
+### Common usages and options for `git push`
+
+* `git push -f`: description
+* `git push -u origin [branch]`: description
+* `git push --all`: description
+* `git push --tags`: description
```
Comment (parkerbxyz) [inline git-guides/git-push.md]:
```suggestion
* `git push -f`: description
* `git push -u origin [branch]`: description
* `git push --all`: description
* `git push --tags`: Publish tags that aren't yet in the remote repository
```

I'm not sure if `description` here is a placeholder. Do each of these need a description to be added?

File: git-guides/git-remote.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+git remote -v
+```
+
+`git remote` manages the set of remotes that you are tracking with your local repository.
+
+### Common `git remote` commands
+
+- `git remote -v`: List the current remotes associated with the local repository
+- `git remote add [name] [URL]`:
+- `git remote remove [name]`:
```
Comment (parkerbxyz) [inline git-guides/git-remote.md]:
```suggestion
- `git remote -v`: List the current remotes associated with the local repository
- `git remote add [name] [URL]`: Add a remote
- `git remote remove [name]`: Remove a remote
```

File: git-guides/git-remote.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+You may need or want to work with multiple remotes for one local repository. This can be common in open source, when a contributor needs to create a fork of a repository to have permission to push changes to the remote.
+
+In this case, it's common to create and clone a fork. Then, the default remote would be `origin`, in reference to the fork. To make it easier to pull any changes to update the local copy of the fork from the original repository, many people add the original repository as a remote also. It's typical to name this remote `upstream`.
+
+![](https://githubtraining.github.io/training-manual/img/fork-to.png)
+
+## Communicating with the remote
+
+There are four commands within Git that prompt communication with the remote. Unless you are using one of these four commands, then all of your work is only happening locally.
```
Comment (parkerbxyz) [inline git-guides/git-remote.md]:
```suggestion
There are four commands within Git that prompt communication with the remote. Unless you are using one of these four commands, all of your work is only happening locally.
```

File: git-guides/git-status.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## What Does `git status` Do?
+
+When in doubt, run `git status`. This is _always_ a good idea. The `git status` command only outputs information, it won't modify commits or changes in your local repository.
+
+A useful feature of `git status` is that it will provide helpful information depending on your current situation. In general, you can count on it to tell you:
+
+- Where `HEAD` is pointing, whether that is a branch or a commit (this is where you are "checked out" to)
+- If you have any changed files in your current directory that have not yet been committed
+- If changed files are un-staged or staged
```
Comment (parkerbxyz) [inline git-guides/git-status.md]:
```suggestion
- If changed files are staged or not
```

This more closely matches Git's terminology for unstaged changes: `Changes not staged for commit`

File: git-guides/git-status.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- If your current local branch is linked to a remote branch, then `git status` will tell you if your local branch is behind or ahead by any commits
+
+During merge conflicts, `git status` will also tell you exactly which files are the source of the conflict.
+
+## How to Use `git status`
+
+### Common usages and options for `git status`
+
+* `git status`: Most often used in its default form, this shows a good base of information
+* `git status -v`: Shows more "verbose" detail including the textual changes of any uncommitted files
```
Comment (parkerbxyz) [inline git-guides/git-status.md]:
```suggestion
- `git status`: Most often used in its default form, this shows a good base of information
- `git status -s`: Give output in short format
- `git status -v`: Shows more "verbose" detail including the textual changes of any uncommitted files
```

File: git-guides/git-push.md (hunk 2 above)
Comment (brianamarie) [reply git-guides/git-push.md]: Good catch! 🙈 Yes, those are all placeholders.

File: git-guides/git-remote.md (hunk 1 above)
Comment (brianamarie) [reply git-guides/git-remote.md]: This is in progress! 🎉

**Comment by brianamarie** [comment]
Thank you all for the reviews and suggestions! 🎉 The image for the fork is in progress. Once we've got that, are we good to ship? 🚢 

cc @tobiasahlin

**Comment by brianamarie** [comment]
image

**Pull request merged**